CHANGELOG.md

```diff
@@ -35,6 +35,7 @@ Same changes as in `@mui/x-data-grid@7.20.1`, plus:
 
 - [DataGridPro] Fix indeterminate checkbox state for server-side data (#14956) @MBilalShafi
 - [DataGridPro] Fix scrolling performance when `rowHeight={undefined}` (#14983) @cherniavskii
+- [DataGridPro] List View (#14393) @KenanYusuf @cherniavskii
```

Comment:
That's a short PR title :laughing:

Reply:
does this require minor bump for the release?

Reply:
We don't announce it publicly yet, we still need docs

Reply:
But a minor bump is still a good idea, I think I'll do that